src/main/kotlin/com/exactpro/th2/check1/rule/AbstractCheckTask.kt

```diff
@@ -99,6 +106,7 @@ abstract class AbstractCheckTask(
         BEGIN,
         TIMEOUT,
         COMPLETED,
+        INTERNAL_ERROR,
```

Comment:
I think we should not add any new states for handling the internal error. If we got an error during initialization we should complete the rule immediately. The initialization happens in the `begin` method. So, I think we can catch the exception that is thrown during calling `.subscribe(this)` at line 255. If any exception is caught we should add information about the cause to the event, send the event and stop the rule execution